hoomd/md/TwoStepNPTMTK.h

```diff
@@ -86,18 +86,12 @@ class PYBIND11_EXPORT TwoStepNPTMTK : public IntegrationMethodTwoStep
             m_T = T;
             }
 
-    //! Update the stress components
-    /*! \param S list of stress components: [xx, yy, zz, yz, xz, xy]
-     */
-    virtual void setS(pybind11::list S)
+        //! Update the stress components
+        /*! \param S list of stress components: [xx, yy, zz, yz, xz, xy]
+         */
+        virtual void setS(const std::vector<std::shared_ptr<Variant>> S)
```

Comment:
```suggestion
        virtual void setS(const std::vector<std::shared_ptr<Variant>>& S)
```
Pass complex types by const reference to avoid copies.